app/javascript/components/Navigation/MatchLogger/MultiplayerLogger/PlacementSelector.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect, useState } from 'react';
+
+const PlacementSelector = ({ participants }) => {
+  const [placements, setPlacements] = useState([]);
+
+  useEffect(() => {
+    setPlacements(participants.map(({ id }) => id));
+  }, [participants]);
+
+  const updatePlacements = (selectedValue, i) => {
+    const newPlace = Number.parseInt(selectedValue, 10);
+    const newPlacements = [...placements];
+    const toReplace = placements.indexOf(newPlace);
+    const replaceWith = placements[i];
+    newPlacements[i] = newPlace;
+    newPlacements[toReplace] = replaceWith;
+    setPlacements(newPlacements);
+  };
```

Comment:
All this logic does is ensure a player can't be selected in two different placements at the same time... i.e.

1 - Pat
2 - Joe
3 - Zack

If I select "Pat" to be in third, the selects will now update to

1 - Zack
2 - Joe
3 - Pat